frontend/src/main/web/lib/components/SystemGlossary.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import Configs from '../constants/Configs';
+import GlossaryStore from '../stores/GlossaryStore';
+import { PureRenderMixin } from 'react/addons';
+import Actions from '../actions/GlossaryActions';
+import { Button, Input, Icons, Icon, Select } from 'zanata-ui'
+import DataTable from './glossary/DataTable'
+import NewEntryModal from './glossary/NewEntryModal'
+import ImportModal from './glossary/ImportModal'
+import { Loader } from 'zanata-ui'
+import _ from 'lodash';
+import StringUtils from '../utils/StringUtils'
+
+var SystemGlossary = React.createClass({
+  mixins: [PureRenderMixin],
+
+  _init: function() {
```

Comment:
I think we should do without the leading underscores - they just make the code harder to type.